source/blood/src/view.cpp

```diff
@@ -1831,6 +1860,10 @@ void viewPrecacheTiles(void)
         tilePrecacheTile(kSBarNumberArmor2 + i, 0);
         tilePrecacheTile(kSBarNumberArmor3 + i, 0);
     }
+    for (int i = 0; i < 6; i++)
```

Comment:
6 could be a constant HUD_NUMBER_COLORS_COUNT or something? The other init loops in here all use magic numbers I believe, but it is confusing IMO.

Reply:
You're right, I'll add that thank you.